lib/src/test/kotlin/validator/PartiQLTestDataValidator.kt

```diff
@@ -302,62 +302,35 @@ class PartiQLTestDataValidator {
 
     // Negative tests
     @Test
-    fun testWrongAnnotationPassParse() {
```

Comment:
Tests related to wrong/missing annotations were removed following the removal of annotations in this PR.